bulla-contracts/schema.graphql

```diff
@@ -286,6 +286,52 @@ type InvoiceReconciledEvent implements IEventLog & IPoolTransaction @entity {
   claim: Claim!
 }
 
+type OrderERC20 @entity {
+  id: ID! # orderId
+  orderId: BigInt!
+  expiry: BigInt!
+  signerWallet: Bytes! #address
+  signerToken: Token! #address
+  signerAmount: BigInt!
+  senderWallet: Bytes! #address
+  senderToken: Token! #address
+  senderAmount: BigInt!
+}
+
+interface IBullaSwapEvent {
+  id: ID!
+  order: OrderERC20!
+  eventName: String!
+  blockNumber: BigInt!
+  transactionHash: Bytes!
+  logIndex: BigInt!
+  timestamp: BigInt!
+}
+
+type OrderCreatedEvent implements IEventLog & IBullaSwapEvent @entity {
+  id: ID!
+  order: OrderERC20!
+  sender: Bytes! #address
+  signerWallet: Bytes! #address
+  eventName: String!
+  blockNumber: BigInt!
+  transactionHash: Bytes!
+  logIndex: BigInt!
+  timestamp: BigInt!
+}
+
+type OrderExecutedEvent implements IEventLog & IOrderEvent @entity {
+  id: ID!
+  order: OrderERC20!
+  sender: Bytes! #address
+  recipient: Bytes! #address
```

Comment:
sould we keep same terminology with signerWallet?

Reply:
was thinking the same overnight